semeio/jobs/config_jobs/PYSCAL

```diff
@@ -0,0 +1,23 @@
+STDERR PYSCAL.stderr
+STDOUT PYSCAL.stdout
+
+TARGET_FILE PYSCAL.OK
```

Comment:
Do we really need a target file for this job? Are we unable to detect failures and just exit with a non-zero exit code?